cdap-spark-core/src/main/scala/co/cask/cdap/app/runtime/spark/DefaultSparkExecutionContext.scala

```diff
@@ -109,7 +109,7 @@ class DefaultSparkExecutionContext(runtimeContext: SparkRuntimeContext,
     try {
       // If there is a SparkContext, wait for the ApplicationEnd callback
       // This make sure all jobs' transactions are committed/invalidated
-      SparkRuntimeEnv.stop.foreach(sc => applicationEndLatch.await())
+      SparkRuntimeEnv.stop().foreach(sc => applicationEndLatch.await())
```

Comment:
The `()`? Because the `stop` method now has a default argument and in Scala, you can only omit the `()` if a method has no argument.